src/Transports.AspNetCore/GraphQLHttpMiddleware.cs

```diff
@@ -101,7 +102,12 @@ public async Task InvokeAsync(HttpContext context)
                         break;
 
                     default:
-                        await WriteErrorResponseAsync(httpResponse, writer, cancellationToken, $"Invalid 'Content-Type' header: non-supported media type. Must be of '{MediaType.JSON}', '{MediaType.GRAPH_QL}' or '{MediaType.FORM}'. {DOCS_URL}");
+                        await WriteErrorResponseAsync(
+                            httpResponse,
+                            writer,
+                            cancellationToken,
+                            $"Invalid 'Content-Type' header: non-supported media type. Must be of '{MediaType.JSON}', '{MediaType.GRAPH_QL}' or '{MediaType.FORM}'. {DOCS_URL}",
+                            415); // UnsupportedMediaType
```

Comment:
Let's change `httpStatusCode` argument type from `WriteErrorResponseAsync` to `System.Net.HttpStatusCode.BadRequest` and pass enum so no comment is needed. Then cast argument to `int` inside.

Reply:
I've change the status code int to a HttpStatusCode.